docs/new_stemcell_line.md

```
@@ -1,30 +1,37 @@
-1. Add, commit, and push the new branch.
+2. Add, commit, and push the new branch.
 
+    ```shell
+    git push --set-upstream origin HEAD
     ```
-    git push origin <<BRANCH_NAME>>
-    ```
-
-1. Switch back to master branch
+3. Update `ci/pipelines/vars.yml` with the appropriate values
 
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)